### ISSUE TITLE
PWCNet comparison experiment

### ISSUE THREAD

**Issue by Prinsphield** [gengshan-y/VCN#8]
Hi Gengshan,

If I change the model to PWCNet as you provided in the repo and run the code under the same training scheme, can I get the same result in the this paper or the original paper in PWCNet?

Thanks~

**Comment by gengshan-y** [comment]
You also have to change some hyperparameters according to our [supplement Sec 6.1](http://www.contrib.andrew.cmu.edu/~gengshay/wordpress/wp-content/uploads/2019/11/vcn.pdf). For example, change learning rate to 1e-4, otherwise the weights will explode.

After changing the hyperparameters, I was able to reproduce the result of PWCNet trained on Chairs and things.
